### PR TITLE
Add weight quantization in post_training_quanzitaion, test=develop

### PR DESCRIPTION
**Pull request by juncaipeng**
In order to reduce the size of model, this api quantizes the weight of some ops from float32 to int8/16. In the inference stage, the quantized weight will be dequantized to float32 again.

### CODE REVIEW

File: python/paddle/fluid/contrib/slim/quantization/post_training_quantization.py [hunk 1]
```diff
@@ -566,3 +574,132 @@ def _safe_entropy(self, reference_distr_P, P_sum, candidate_distr_Q, Q_sum):
...
+                parameters were saved in a single binary file.
+            quantizable_op_type(list[str], optional): The list of ops 
+                that will be quantized, and the quantized ops should be
+                contained in ["conv2d", "depthwise_conv2d", "mul"]. 
+                Default is ["conv2d","mul"].
+            quantize_weight_bits(int, optional): The bits for the quantized
+                weight, and it should be 8 or 16. Default is 8.
+            threshold_rate(float, optional): This api uses abs_max methd to 
+                quantize the weight from float32 to int8/16, and the abs max 
+                value is important for quantization diff. When the is far
```
Comment (wzzju) [inline python/paddle/fluid/contrib/slim/quantization/post_training_quantization.py]: `When the is far`?

**Review by wzzju — DISMISSED** [review]
LGTM.

File: python/paddle/fluid/contrib/slim/quantization/post_training_quantization.py (hunk 1 above)
Comment (juncaipeng) [reply python/paddle/fluid/contrib/slim/quantization/post_training_quantization.py]: Done

**Review by wzzju — APPROVED** [review]
LGTM.

**Comment by tangtang586** [comment]
this op name mentioned in the title should be post_training_quantization.

**Pull request merged**